web/settings/modulconfiglp.php

```diff
@@ -1222,7 +1222,67 @@ function getCookie(cname) {
 											<input class="form-control" type="text" name="mypeugeot_clientsecretlp1" id="mypeugeot_clientsecretlp1" value="<?php echo $mypeugeot_clientsecretlp1old ?>">
 										</div>
 									</div>
+									<div class="form-row mb-1">
+									<label class="col-md-4 col-form-label">Kombiniere Peugeot SoC Modul und manuelle Berechnung</label>
+										<div class="col">
+											<div class="btn-group btn-group-toggle btn-block" data-toggle="buttons">
+												<label class="btn btn-outline-info<?php if($mypeugeot_soccalclp1old == 0) echo " active" ?>">
+													<input type="radio" name="mypeugeot_soccalclp1" id="mypeugeot_soccalclp1Off" value="0"<?php if($mypeugeot_soccalclp1old == 0) echo " checked=\"checked\"" ?>>Nein
```

Comment:
Die neue Einstellung `mypeugeot_soccalclp1` (und auch für `...lp2`) müssen in der `openwb.conf` eingetragen werden. Solche Änderungen werden durch die `runs/atreboot.sh` durchgeführt, die nach einem Update und beim Neustart ausgeführt wird. Bitte dort analog zu den vorherigen Zeilen in der `atreboot.sh` einfügen: https://github.com/snaptec/openWB/blob/dfd9a5d17f3e4856f20df73ec99172e72c7776e0/runs/atreboot.sh#L2810